README.md

```diff
@@ -16,16 +16,16 @@ IN-CORE consists of multiple components as shown below:
 
 ## Current Release
 
-This is the list of releases for IN-CORE v4.0.0
+This is the list of releases for IN-CORE v4.1.0
 
 | module | version |
 | ------ | ------- |
 | incore-auth | 1.6.0 |
-| incore-docs | v4.0.0 |
-| incore-helm | 1.15.0 |
+| incore-docs | v4.1.0 |
+| incore-helm | 1.16.0 |
 | incore-lab | v0.5.4 |
```

Comment:
incore-lab is 0.6.0... However, maybe we need to separate between incore-lab and incore-hub